lib/get-format.js

```diff
@@ -60,8 +66,11 @@ function getDataProtocolModuleFormat(parsed) {
 
 /**
  * @param {URL} url
+ * @param {{parentURL: string}} _context
+ * @param {boolean} ignoreErrors
+ * @returns {string|null}
```

Comment:
```suggestion
 * @type {ProtocolHandler}
```

Now that the function is defined as a type, it can be reused here